File: backend/src/Core/Dev4All.Domain/Enums/ContractStatus.cs
namespace Dev4All.Domain.Enums;

public enum ContractStatus
{
    Draft = 0,          // Sistem taslağı oluşturdu
    UnderReview = 1,    // En az bir taraf revize etti
    BothApproved = 2,   // İki taraf da onayladı → Ongoing'e geçiş
    Cancelled = 3       // Bir taraf iptal etti → proje Cancelled
}

File: mobile/core/domain/src/main/kotlin/com/dev4all/mobile/core/domain/model/ContractStatus.kt
package com.dev4all.mobile.core.domain.model

/**
 * Contract lifecycle states — matches backend Dev4All.Domain.Enums.ContractStatus.
 *
 * Draft(0) → UnderReview(1) → BothApproved(2) / Cancelled(3)
 */
enum class ContractStatus {
    Draft,
    UnderReview,
    BothApproved,
    Cancelled,
}
